common/src/main/java/me/confuser/banmanager/BmAPI.java

```diff
@@ -35,17 +35,6 @@ public static PlayerData getPlayer(UUID uuid) throws SQLException {
     return plugin.getPlayerStorage().queryForId(UUIDUtils.toBytes(uuid));
   }
 
-  /**
-   * @param player
-   *
-   * @return PlayerData
-   *
-   * @throws SQLException
-   */
-  public static PlayerData getPlayer(Player player) throws SQLException {
```

Comment:
@clienthax what's the reason for removing these?

Reply:
Bukkit Player class, can't have it in the API as its implementation specific.
If we wan't to keep it we could delegate down to the other handlers, but it seems easier to just scrap it off completly.